src/databricks/labs/ucx/recon/schema_comparator.py

```diff
@@ -26,8 +34,12 @@ def compare_schema(self, source: TableIdentifier, target: TableIdentifier) -> Sc
         return SchemaComparisonResult(is_matching, comparison_result)
 
     def _eval_schema_diffs(self, source: TableIdentifier, target: TableIdentifier) -> list[SchemaComparisonEntry]:
-        source_metadata = self._metadata_retriever.get_metadata(source)
-        target_metadata = self._metadata_retriever.get_metadata(target)
+        source_metadata = self._metadata_retriever.get_metadata(
+            source, column_name_transformer=self._column_name_transformer()
+        )
+        target_metadata = self._metadata_retriever.get_metadata(
+            target, column_name_transformer=self._column_name_transformer()
+        )
```

Comment:
Does not really matter, but my OCD likes to initialize the transformer once. Also, I prefer the full names, but would accept a shorter name within a small code block to increase readability with all statements being single lines

```suggestion
        transformer = self._column_name_transformer()
        source_metadata = self._metadata_retriever.get_metadata(source, column_name_transformer=transformer)
        target_metadata = self._metadata_retriever.get_metadata(target, column_name_transformer=transformer)
```